lib/ript/dsl/primitives/filter.rb

```diff
@@ -80,7 +79,6 @@ def build_rule(name, block, opts={})
               to_address    = @labels[to][:address]
 
               attributes = {
-                             "table"       => "filter",
```

Comment:
Yes sir, I think we have hit a weird bug in iptables somewhere Ill add the filter back in and post the results..